static/js/services/live-list.js

```diff
@@ -108,12 +109,15 @@ angular.module('inboxServices').factory('LiveListConfig',
           scope.lineage = report.lineage;
           scope.unread = !report.read;
           var element = renderTemplate(scope);
+          element = $(element);
           if (removedDomElement &&
               removedDomElement.find('input[type="checkbox"]').is(':checked')) {
             // updating an item that was selected in select mode
-            element = $(element);
             element.find('input[type="checkbox"]').prop('checked', true);
           }
+          // So that relative-date directive is rendered inside a 'content_row_list_item' template.
+          $compile(element.contents())(scope);
```

Comment:
I've spent quite a lot of time on this. Had to open a [fiddle](http://jsfiddle.net/cbrwizard/4e2r2o07/) to understand that it's not Angular's fault hehe. Without this all directives inside `content_row_list_item` don't get rendered.

Reply:
We had some serious performance issues using angular for the lists so we essentially wrote our own renderer with jquery. We're planning to migrate off angularjs to angular/react/something soon and my hope is that will be sufficiently fast so we can use it for lists too.

I think your change will reintroduce the performance issue because it's going back to using angular bindings (if I understand it correctly). LiveList already has custom code to update the dates so it's the one place where your change isn't required. Can you revert this and leave LiveList working as it did before?